storybook/storybook-react/src/LinkList/LinkList.stories.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2024 Gemeente Amsterdam
+ */
+
+import { LinkList } from '@amsterdam/design-system-react'
+import { ChattingIcon, HousingIcon, PhoneIcon } from '@amsterdam/design-system-react-icons'
+import { useArgs } from '@storybook/preview-api'
+import { Meta, StoryObj } from '@storybook/react'
+import { exampleLinkList } from '../shared/exampleContent'
+
+const links = exampleLinkList()
+
+const meta = {
+  title: 'Navigation/Link List',
+  component: LinkList,
+  args: {
+    children: 'Default children',
+  },
+  argTypes: {
+    children: {
+      table: { disable: true },
+    },
+  },
+} satisfies Meta<typeof LinkList>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  render: function Render() {
+    const [args] = useArgs()
+
+    return (
+      <LinkList {...args}>
+        {links.map((text, index) => (
+          <LinkList.Link key={index} href="#" size={args['size']}>
+            {text}
+          </LinkList.Link>
+        ))}
+      </LinkList>
+    )
+  },
```

Comment:
This seems to break the Show code window

Reply:
The stories have gotten a big update.